### PR TITLE
ci: run build image on self-hosted arm machine

### PR DESCRIPTION
**Pull request by YangKeao**
Signed-off-by: YangKeao <yangkeao@chunibyo.icu>

### What problem does this PR solve?

Run build image ci on self-hosted arm machine

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [x] release-2.2
  - [x] release-2.1

### CODE REVIEW

**Comment by YangKeao** [comment]
@STRRL @Hexilee PTAL

File: .github/workflows/upload_image.yml [hunk 1]
```diff
@@ -46,16 +46,13 @@ jobs:
...
-          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
-
           if [ "${IMAGE}" = "chaos-dashboard" ]; then
             UI=1
           else
             UI=0
           fi
           # ${VAR,,} convert VAR to lower case
           make -B \
-            TARGET_PLATFORM=$ARCH \
```
Comment (STRRL) [inline .github/workflows/upload_image.yml:58]: Could we keep this line?

Comment (YangKeao) [reply .github/workflows/upload_image.yml]: I think we can :thinking:

Comment (YangKeao) [reply .github/workflows/upload_image.yml]: Keep!

**Review by STRRL — APPROVED** [review]
LGTM!

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**